static/js/settings_account.js

```diff
@@ -501,7 +502,10 @@ export function set_up() {
                         4000,
                     );
                 }
-                overlays.close_modal("#change_email_modal");
+                dialog_widget.close_modal();
+            },
+            error_continuation() {
+                change_email_error.show();
```

Comment:
I am not sure this is required. It should work correctly even if we remove this since ui_report handles showing the error. Can you verify this?